tests/enrichment/address_validation_mapping.json

```diff
@@ -0,0 +1,5 @@
+{"input_address": "66 Church St Cambridge Mass", "validated_formatted_address": "66 Church Street, Cambridge, MA 02138-3733, USA", "expiration": "2023-08-09 11:46:40.408657", "region_code": "US", "postal_code": "02138-3733", "admin_area": "MA", "locality": "Cambridge", "address_lines": ["66 Church St"], "usps_first_address_line": "66 CHURCH ST", "usps_city_state_zip_line": "CAMBRIDGE MA 02138-3733", "usps_city": "CAMBRIDGE", "usps_state": "MA", "usps_zip_code": "02138-3733", "latitude": 42.3739503, "longitude": -71.1211445, "place_id": "ChIJNR2ZIGh344kRNQAj-dh6d00", "input_granularity": "PREMISE", "validation_granularity": "PREMISE", "geocode_granularity": "PREMISE", "has_inferred": true, "has_unconfirmed": false, "has_replaced": false, "address_complete": false}
```

Comment:
Do you need these files as part of the PR and testing? They will be added to the package if included in the present form. I know the addresses are not that critical but still, I am wondering if we can test without adding these files to the PR. I don't see equivalent files for translation either...

Reply:
I may have missed it but where do you use these 2 json files?

Reply:
These are used to test the from_json functions and from_json error handling for module. The translation doesn't have them because it has data in the test file, which it saves to json and then loads, and it doesn't test the same error handling. We could get rid of them by doing a save + load in the test file and mocking the bad data somehow, but the same data would still be stored somewhere.